showcase/src/assets/translations/en.json

```diff
@@ -236,6 +236,20 @@
         "TITLE": "Stark Typography"
       }
     },
+    "HOMEPAGE": {
+      "MAIN_TITLE": "About Stark",
+      "STARK_DESC_BEFORE_DETAIL": "Stark is an Angular 5+ based front-end framework built on top of Angular Stark features... An awesome reusable build using Webpack, built-in support for state of the art front-end tech. Stark provides main building blocks for accelerating front-end development:",
+      "STARK_DESC_DETAIL": "<ul><li>a solid reusable build based on Webpack</li><li>a starter project inspired by Angular Starter by AngularClass and Google's Web Starterk Kit</li><li>core modules providing useful APIs</li><li>UI modules providing interesting UI components </li><li>a showcase application containing: <ul><li>our living style guide</li><li>a showcase of all components/services along with their API</li><li>our developer guide</li></ul></li></ul>",
+      "STARK_DESC_AFTER_DETAIL": "Stark modules are like LEGO blocks: add what you need, no less, no more. If you don't like our defaults then you may replace/override/ignore anything you fancy.<br> <a href=\"https://github.com/NationalBankBelgium\"><img class=\"github-link\" src=\"/assets/images/logo/github.png\"></a> And even better, the Stark repository is available on the National Bank of Belgium page on Github!<br>Come and have a look at our amazing open sources projects!",
+      "STARK_NEWS_TITLE": "Stark News",
+      "STARK_NEWS_DESCRIPTION": "Want to find the hottest news about the Stark framework?",
+      "STARK_NEWS_CONTENT": "Have a look at the news page right <a href=\"news\">here!</a>",
+      "STARK_DOC_MAIN_TITLE": "Stark Documentation",
+      "STARK_DOC_MAIN_DESC": "Want some more info about Stark? Take a look at our links just below!",
+      "STARK_DOC_CORE_DESC": "Stark Core, aka stark-core, is a module providing reusable APIs, such as a routing, logging, log shipping, etc.<br>You can find the latest API documentation for stark-core <a href=\"https://stark.nbb.be/api-docs/stark-core/latest\">here</a>.",
```

Comment:
At the end of this text, you can add something like: "Or check the API for previous releases" and add the link to all the docs: https://github.com/NationalBankBelgium/stark/tree/gh-pages/api-docs/stark-core